scripts/config-mock-build.sh

```diff
@@ -0,0 +1,6 @@
+# copy build to new folder and replace all usages of :8001 with appropriate mock port
+BUILD_PATH=./build/test/mock-builds/port-$2-$(openssl rand -hex 12)
+mkdir -p $BUILD_PATH
+cp -r ./$1/* $BUILD_PATH
+grep ":8001" $BUILD_PATH -lr | xargs sed -i.bak -e "s/\:8001/\:$2/g"
```

Comment:
Can it be solved with a `[BUILD_REPLACEABLE_....]` placeholder?

Reply:
I suppose it could.. we would still need to do two rounds of replacing? The step that normally replaces `[BUILD_REPLACEABLE_....]` and currently puts the `:8001` would still need to insert some more maybe `[TEST_REPLACEABLE_MOCK_PORT]` and then that would be replaced here.

It's true that as things are now, there is no meaning to the `:8001` in particular. So that should not be put it, to not be confusing. So `[TEST_REPLACEABLE_MOCK_PORT]` or similar would be a better choice here.